src/snowflake/snowpark/modin/plugin/docstrings/io.py

```diff
@@ -343,6 +343,125 @@ def read_xml():
 def json_normalize():
     """
     Normalize semi-structured JSON data into a flat table.
+
+    Parameters
+    ----------
+    data : dict or list of dicts
+        Unserialized JSON objects.
+    record_path : str or list of str, default None
+        Path in each object to list of records. If not passed, data will be assumed to be an array of records.
+    meta : list of paths (str or list of str), default None
+        Fields to use as metadata for each record in resulting table.
+    meta_prefix : str, default None
+        If True, prefix records with dotted (?) path, e.g. foo.bar.field if meta is [‘foo’, ‘bar’].
```

Comment:
Do we know why the question marks are here? seems like an artifact or typo in the original docs that we don't need to replicate.

Reply:
Updated.